roles/foreman_repositories/molecule/debian/prepare.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: prepare
+  hosts: all
+  gather_facts: true
+  tasks:
+    - name: install gpg
```

Comment:
kinda duplicates what's in `roles/foreman_repositories/tasks/debian.yml`?

Reply:
do you only need the `update_cache` part for prepare in a container?

Reply:
Ahh yea, that explains things. Yes, I needed the update_cache, I can drop that here and add it to the task itself.

Reply:
didn't you want to make this go away? :)